### PR TITLE
🌱  Update compatibility in README.md

### PR DESCRIPTION
**Pull request by sbueringer**
Signed-off-by: Stefan Büringer buringerst@vmware.com

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,18 +31,19 @@ Check out the [getting started guide](./docs/getting_started.md) for launching a
...
+| CAPV v1beta1 (v1.4)              |              ☓              |              ☓              |             ☓              |             ✓              |             ☓              |             ☓              |
+| CAPV v1beta1 (v1.5)              |              ☓              |              ☓              |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPV v1beta1 (v1.6)              |              ☓              |              ☓              |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPV v1beta1 (v1.7)              |              ☓              |              ☓              |             ☓              |             ✓              |             ✓              |             ☓              |
+| CAPV v1beta1 (v1.8)              |              ☓              |              ☓              |             ☓              |             ☓              |             ☓              |             ✓              |
```
Comment (sbueringer) [inline README.md:46]: I don't know how this was filled out before, but I would stick with supporting what we test which is the combination CAPV v1.8 & CAPI v1.5

We should probably rethink the format of this table as it doesn't scale well, but let's keep it as is for now

File: README.md [hunk 2]
```diff
@@ -31,18 +31,19 @@ Check out the [getting started guide](./docs/getting_started.md) for launching a
...
-| CAPV v1beta1 (v1.5) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
-| CAPV v1beta1 (v1.6) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
-| CAPV v1beta1 (v1.7) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
+|                                  | Cluster API v1alpha3 (v0.7) | Cluster API v1alpha4 (v0.8) | Cluster API v1beta1 (v1.0) | Cluster API v1beta1 (v1.3) | Cluster API v1beta1 (v1.4) | Cluster API v1beta1 (v1.5) |
+|----------------------------------|:---------------------------:|:---------------------------:|:--------------------------:|:--------------------------:|:--------------------------:|:--------------------------:|
+| CAPV v1alpha3 (v0.7)(deprecated) |              ✓              |              ✓              |             ✓              |             ✓              |             ☓              |             ☓              |
```
Comment (killianmuldoon) [inline README.md:36]: WDYT about getting rid of the deprecated versions at least for now?

File: README.md [hunk 3]
```diff
@@ -31,18 +31,12 @@ Check out the [getting started guide](./docs/getting_started.md) for launching a
...
-| CAPV v1beta1 (v1.3, master) |              ☓              |              ☓              |             ☓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.4) |              ☓              |              ☓              |             ☓              |             ✓              |             ☓             |
-| CAPV v1beta1 (v1.5) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
-| CAPV v1beta1 (v1.6) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
-| CAPV v1beta1 (v1.7) |              ☓              |              ☓              |             ☓              |             ✓              |             ✓             |
+|                                  | Cluster API v1alpha3 (v0.7) | Cluster API v1alpha4 (v0.8) | Cluster API v1beta1 (v1.0) | Cluster API v1beta1 (v1.3) | Cluster API v1beta1 (v1.4) | Cluster API v1beta1 (v1.5) |
```
Comment (killianmuldoon) [inline README.md:34]: Could we also get rid of v1alpha3 and v1alpha4 from the top bar? Should make the whole thing easier to read.

Comment (sbueringer) [reply README.md:34]: Oh of course ups

Comment (sbueringer) [reply README.md]: Done, thx for catching it

Now the table looks okay

**Review by killianmuldoon — COMMENTED** [review]
/lgtm

**Pull request merged**